### PR TITLE
Mentor guide

### PR DESCRIPTION
**Pull request by aliok**
Fixes https://github.com/cncf/mentoring/issues/736

cc @Riaankl 
I see there's more in https://github.com/cncf/mentoring/issues/736 like instructions about program specific things, but I didn't write them. ...well, I can't write them anyway as I don't know how the platforms work other than GSoC.

IMO, we should keep our mentor guide generic. Platform related instructions are sent with each announcement anyway (like, "It is time to review mentee applications. Use this page / that form to inform us with your selection", etc.)

### CODE REVIEW

File: mentors/README.md [hunk 1]
```diff
@@ -1,78 +1,127 @@
...
+*  Mentee applications close ([announcement](https://github.com/cncf/mentoring/discussions/918))
+  * Mentors review the proposals and provide feedback
+* Accepted mentees are [announced](https://github.com/cncf/mentoring/discussions/954)
+  * Mentees bond with the community and the mentors
+  * Mentors and mentees start working on their projects
+* TBA: (we're in the middle of the program)
 
+## Success Stories
 
+TBD
```
Comment (aliok) [inline mentors/README.md:68]: Any mentors you know you would like to share a 2 paragraph story? I think the story should tell the audience about the rewarding experience of a mentor and a mentee, but the focus should be on the mentor's experience as the audience of this doc is mentors.

I can immediately think of @bradmccoydev , if he's interested.

File: mentors/README.md [hunk 2]
```diff
@@ -1,78 +1,127 @@
...
+  * Mentors send their project idea PRs ([example](https://github.com/cncf/mentoring/pull/810))
+* Mentee applications start ([announcement](https://github.com/cncf/mentoring/discussions/892))
+  * Mentee candidates interact with the community and the mentors
+  * Mentee candidates submit their proposals to the GSoC platform
+*  Mentee applications close ([announcement](https://github.com/cncf/mentoring/discussions/918))
+  * Mentors review the proposals and provide feedback
+* Accepted mentees are [announced](https://github.com/cncf/mentoring/discussions/954)
+  * Mentees bond with the community and the mentors
+  * Mentors and mentees start working on their projects
+* TBA: (we're in the middle of the program)
```
Comment (aliok) [inline mentors/README.md:64]: I am curious about your opinions on this section. What I wanted to provide is a realistic view of the "paperwork" (submiting project idea, reviewing mentee proposals, etc).

**Comment by Riaankl** [comment]
@aliok This great work.
I see a few things that I would surgest changes for. Would be easier to collaberate on a google doc and copy the results back to the PR.
What do you think @nate-double-u ?

**Comment by nate-double-u** [comment]
> Would be easier to collaberate on a google doc and copy the results back to the PR. What do you think @nate-double-u ?

At this point it's probably best to have the discussion in public -- I'm a fan of drafting PRs in google docs or in hackmd, but once the PR is made then splitting the conversation around edits and suggestions can make things harder.

The GitHub UI does allow for `suggestions` to be [applied across multiples lines](https://github.blog/changelog/2020-04-15-multi-line-code-suggestions-general-availability/), which makes comments on large PRs like this a little easier.

File: mentors/README.md [hunk 3]
```diff
@@ -1,78 +1,127 @@
...
-1. I didn't finish a program in the past. Can I still take part in mentorship opportunities?
-1. Can I apply for the LFX Mentorship again after successfully completing a Semester?
-1. I've submitted my application or requested information and haven't heard back by the expected timeframe. How can I follow up?
+## Support
 
+Mentors have a direct line to us!
 
+If you have any questions, please reach out to the CNCF mentoring group on [CNCF Slack](https://cloud-native.slack.com/archives/CGPK98JNQ) or on [CNCF Mentoring repository discussions](https://github.com/cncf/mentoring/discussions). We are happy to help you mentor your mentees!
 
+## Mentor Resources
```
Comment (aliok) [inline mentors/README.md:103]: Need to link https://docs.linuxfoundation.org/lfx/mentorship/mentor-guide/getting-started/what-makes-a-good-mentor as a generic guide regardless of the program

**Comment by aliok** [comment]
Created https://docs.google.com/document/d/1Zozg3g5qE4jkdVEdiVtCoXvc_iuuOls1hfCgW6Pf7jQ/edit?usp=sharing

Closing this PR now. Let's collaborate on the Google Doc and I will create a new PR later.

File: mentors/README.md (hunk 1 above)
Comment (bradmccoydev) [reply mentors/README.md:68]: Hey Ali, sorry i was super busy this week and it took awhile to respond. I will put on my list to create a story so you can share in the future if you want.

Comment (aliok) [reply mentors/README.md:68]: Thanks @bradmccoydev . We will merge w/o the success stories, but your story would be helpful. Thanks! No rush BTW.

**Pull request closed without merging**